code/DDSCodeTester.cpp

```diff
@@ -1185,6 +1186,34 @@ void dds_discovery_examples()
             return;
         }
         //!--
+
+        // Check XML static discovery from file
+        std::string file = "file://static_Discovery.xml";// The (file://) flag it is optional.
+        DomainParticipantFactory* factory = DomainParticipantFactory::get_instance();
+        if (factory->check_xml_static_discovery(file) != ReturnCode_t::RETCODE_OK)
```

Comment:
```suggestion
        if (ReturnCode_t::RETCODE_OK != factory->check_xml_static_discovery(file))
```

We usually try to keep the constant expression which is checked in the left hand side